portfolio/src/main/java/com/google/sps/data/BlogHashMap.java

```diff
@@ -25,14 +25,13 @@
 
 public final class BlogHashMap {
   private final int DEFAULT_LOAD_AMOUNT = 10;
-  private final InternalTags internal_tags;
   private Map<String, LinkedList<BlogMessage>> map;
 
   // Takes and puts a list of BlogMessage type in map.
   // Each BlogMessage type should contain tag that matches tags in the internal System. 
   public void putInMap(List<BlogMessage> messages) {
     for (BlogMessage message : messages) {
-      if (internal_tags.tagIsSupported(message)) {
+      if (InternalTags.tagIsSupported(message)) {
```

Comment:
Would checking the opposite make the nesting better, e.g.:

if (!InternalTags.tagIsSupported(message)) {
  continue;
}
// Process the case with supported tag with 1 fewer level of nesting.